spec/resources/api/v2/transfer_template_resource_spec.rb

```diff
@@ -8,28 +8,10 @@
 
   let(:resource_model) { build_stubbed :transfer_template }
 
-  # Test attributes
-  it 'has the expected attributes', :aggregate_failures do
-    expect(resource).not_to have_attribute :id
-    expect(resource).to have_attribute :uuid
-    expect(resource).to have_attribute :name
-  end
-
-  # Updatable fields
-  it 'allows updating of read-write fields', :aggregate_failures do
-    expect(resource).to have_updatable_field :name
-  end
-
-  it 'disallows updating of read-only fields', :aggregate_failures do
-    expect(resource).not_to have_updatable_field :uuid
-  end
+  # Attributes
+  it { is_expected.to have_readonly_attribute :uuid }
+  it { is_expected.to have_readwrite_attribute :name }
 
   # Filters
   it { is_expected.to filter(:uuid) }
-
```

Comment:
can this be removed?

Reply:
The check for the filter?  The resource does contain the filter definition, so we'd have to remove that as well.  I don't think we should do that.